# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../netpbm/pnmtopng-zlib.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

http://bugs.gentoo.org/291987

--- netpbm-10.35.74/converter/other/Makefile.orig       2009-09-18 21:04:37.000000000 +0200
+++ netpbm-10.35.74/converter/other/Makefile    2010-05-28 18:09:51.000000000 +0200
@@ -159,6 +159,7 @@
          $(shell $(LIBOPT) $(NETPBMLIB)) \
          $(PNGLIB_LIBOPTS) $(MATHLIB) $(LDFLAGS) $(LDLIBS) $(RPATH) $(LADD)

+pnmtopng: LADD += -lz # pnmtopng calls zlib_version itself
 pnmtopng: %: %.o pngtxt.o $(NETPBMLIB) $(LIBOPT)
        $(LD) -o $@ $@.o pngtxt.o \
          $(shell $(LIBOPT) $(NETPBMLIB)) \
